package.json

```
@@ -29,8 +29,8 @@
   "scripts": {
     "setup": "yarn install && yarn allow-scripts",
     "build": "tsc",
-    "test": "testem ci -P 2",
-    "buildTest": "yarn build && browserify test/index.js -o test/bundle.js",
+    "test": "yarn buildTest && playwright test",
```

This might be enough to avoid having a HTML page
```suggestion
    "test": "yarn buildTest && playwright test test/bundle.js",
```

---

I was only able to get it to run with a webpage. It does not accept a JS bundle via CLI like this.

I even tried using something like this in place of a page: https://playwright.dev/docs/api/class-frame#frame-add-script-tag

But that failed as well; seemingly until you navigate to a "real" page once, it is unable to access `window.crypto`. Maybe it starts on the `about:blank` page and you can't access certain APIs from there? Unsure.